src/main.js

```diff
@@ -1,3 +1,4 @@
-const placeholder = {};
+export { default as Label } from './components/Label';
 
-export default placeholder;
+export { default as AbsoluteLayout } from './layouts/AbsoluteLayout';
+export { default as StackLayout } from './layouts/StackLayout';
```

Comment:
no need to export StackLayout component within this pull request

Reply:
Removed StackLayout from main.js